riva/clients/tts/riva_tts_client.cc

```diff
@@ -242,6 +250,10 @@ main(int argc, char** argv)
       ::riva::utils::wav::Write(FLAGS_audio_file, rate, pcm.data(), pcm.size());
     }
   } else {  // online inference
+    if (is_a2flow) {
+      LOG(ERROR) << "A2Flow model does not support online inferencing.";
```

Comment:
better to remove a2flow from the log here, we can say that prompt transcript field is not supported for online inference